pkg/apis/vpc/v1alpha1/vpcnetworkconfiguration_types.go

```diff
@@ -9,9 +9,10 @@ import (
 )
 
 const (
-	AccessModePublic  string = "Public"
-	AccessModePrivate string = "Private"
-	AccessModeProject string = "PrivateTGW"
+	AccessModePublic       string = "Public"
+	AccessModePrivate      string = "Private"
+	AccessModeProject      string = "PrivateTGW"
+	AccessModeProjectInNSX string = "Private_TGW"
```

Comment:
we should remove podSubnetAccessMode in VPCNetworkConfiguration CR.

Reply:
https://github.com/vmware-tanzu/nsx-operator/issues/680
will update in another PR.